piet-coregraphics/src/lib.rs

```diff
@@ -77,6 +77,10 @@ impl<'a> CoreGraphicsContext<'a> {
             let xform = Affine::FLIP_Y * Affine::translate((0.0, -height));
             ctx.concat_ctm(to_cgaffine(xform));
         }
+        // these two options are the default on 10.15; we set them explicitly to ensure
```

Comment:
These don't seem like useful defaults, I think font smoothing should be on.

Reply:
okay, I'll look into this a bit more.

Reply:
okay, I removed this, does seem like it might be a gotcha.